src/components/DeckBuilderDryRunView/index.js

```diff
@@ -10,10 +12,14 @@ export default props => {
   // purposes. The mode is restored to `AUTOMATIC` as soon as the 4th card has
   // been picked.
   const [mode, setMode] = React.useState(params.get('mode') || 'AUTOMATIC')
+  const [modifier, setModifier] = React.useState('0')
   const [equalsMode, setEqualsMode] = React.useState(false)
-  const deck = equalsMode
-    ? props.deck.map(card => ({ ...card, level: 1 }))
-    : props.deck
+  const fullDeck = props.deck.map(card =>
+    equalsMode
+      ? resolveCardForLevel({ ...card, level: 1 })
+      : resolveCardForLevel(card)
```

Comment:
How comes you need a `resolveCardForLevel` here now?

Reply:
The race/type attributes were not yet in the deck

Reply:
It contains only id and level, as added in the DeckBuilderRoot. `resolveCardForLevel` is probably called twice now though - maybe adding these attributes directly in the deckBuilderEditorView would be better

Reply:
Ah yes, that makes a lot of sense; we never needed card info before. I think what you did is good then!